llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -459,7 +460,8 @@ bool RuntimeCheckingPtrGroup::addPointer(unsigned Index, const SCEV *Start,
 }
 
 void RuntimePointerChecking::groupChecks(
-    MemoryDepChecker::DepCandidates &DepCands, bool UseDependencies) {
+    const MemoryDepChecker &DepChecker,
```

Comment:
Just checking if this pull request is still active. Also, it looks like we pass `DepChecker` here, but we do not use it. Is it needed for a future change?

Reply:
Yep will try and update it in the next few days, thanks